operator/pkg/helmreconciler/reconciler.go

```diff
@@ -489,3 +495,53 @@ func (h *HelmReconciler) reportPrunedObjectKind() {
 			Increment()
 	}
 }
+
+// createNamespace creates a namespace using the given k8s client.
+func (h *HelmReconciler) createNamespace(namespace string, network string) error {
```

Comment:
Is it safe to remove `createNamespace` from `shared.go` as it is moved in `reconciler`? 
https://github.com/istio/istio/blob/b0474ef00481d31157bb9f4cab3f8690ef6fb6a9/operator/cmd/mesh/shared.go#L246

Reply:
Looks like the `istioctl operator init -f <cr.yaml>` still needs that function.